Userland/Libraries/LibVideo/Color/CodingIndependentCodePoints.h

```diff
@@ -240,4 +240,17 @@ constexpr StringView matrix_coefficients_to_string(MatrixCoefficients matrix_coe
     return "Reserved"sv;
 };
 
+constexpr StringView video_full_range_flag_to_string(VideoFullRangeFlag video_full_range_flag)
+{
+    switch (video_full_range_flag) {
+    case VideoFullRangeFlag::Studio:
+        return "Studio"sv;
+    case VideoFullRangeFlag::Full:
+        return "Full"sv;
+    case VideoFullRangeFlag::Unspecified: // Not part of the spec, serenity-specific addition for convenience.
+        return "Unspecified"sv;
+    }
+    return "Unknown"sv;
```

Comment:
At least `icc` takes a number from the icc data, casts it to `VideoFullRangeFlag` and passes it to this function. (It's currently the only client of this function, too.) So it can happen, depending on input data, yes.